packages/glimmer-runtime/lib/compiled/opcodes/dom.ts

```diff
@@ -63,18 +62,48 @@ export class OpenPrimitiveElementOpcode extends Opcode {
   }
 }
 
-export class PushRemoteElementOpcode extends Opcode {
+export class UpdateBounds extends UpdatingOpcode {
+  public type = "update-bounds";
+  private cache: ReferenceCache<Opaque>;
+  private bounds: RemoteBlockTracker;
+  private vm: VM;
+
+  constructor(vm: VM, cache: ReferenceCache<Opaque>, bounds: RemoteBlockTracker) {
+    super();
+
+    this.vm = vm;
+    this.cache = cache;
+    this.bounds = bounds;
+  }
+
+  evaluate(vm: UpdatingVM) {
+    let { cache, bounds } = this;
+
+    if (isModified(cache.revalidate())) {
+      bounds.update(cache.peek());
+    }
+  }
+
+  toJSON(): OpcodeJSON {
+    return {
+      guid: this._guid,
+      type: this.type
```

Comment:
I probably am missing things here, but I didn't know enough to know what all should be here.